### DISCUSSION TITLE
Next/Previous Not Working

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#24] [Fixed Issues]
The next/previous buttons do not always jump to the next or previous track. It seems to select a track at random.

**Comment by Sagnasty** [comment]
Is shuffle play active in the playback window?

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Reply by R101G** [reply]
I think so, but there did not seem any way to turn it off. It is a problem which also occurred, and was fixed, in an earlier version. I did a fresh install, and it has not happened since, so I will move this to the fixed category.
